js/ui/form/ui.form.utils.js

```diff
@@ -118,3 +120,37 @@ function _renderLabelMark({ isRequiredMark, requiredMark, isOptionalMark, option
         .addClass(isRequiredMark ? FIELD_ITEM_REQUIRED_MARK_CLASS : FIELD_ITEM_OPTIONAL_MARK_CLASS)
         .text(String.fromCharCode(160) + (isRequiredMark ? requiredMark : optionalMark));
 }
+
+export function renderButton({ buttonOptions, createComponentCallback }) {
+    const $button = $('<div>');
+    createComponentCallback($button, 'dxButton', buttonOptions);
```

Comment:
$button is not in DOM tree, see https://github.com/DevExpress/DevExtreme/pull/18488/files